server/handlers/shared_validators.js

```diff
@@ -145,3 +145,16 @@ exports.walletBodyValidator = body("wallet")
   .notEmpty()
   .bail()
   .isLength({ min: 0 });
+ 
+  exports.checkLoggedClientMiddleware = (req, res, next) => {
+    /* MOCK DATA ... waiting for login */ 
+    var isClientLogged = true; 
+    var loggedClientID = req.body.clientID;   // ClientID of the logged client
+    /* END MOCK DATA */ 
+    
+    if(isClientLogged){
+      if(loggedClientID != req.body.clientID)
```

Comment:
You can't do this check. Not every client restricted api accepts a clientID in the body. Actually, this is the only api that works like this.

Reply:
In fact, I use this middleware only in the CreateOrder API